version/news/NEWS-2024.12.1-kousa-dogwood.md

```diff
@@ -16,6 +16,7 @@
 - Fixed an issue where RStudio could emit a warning when attempting to retrieve help for R objects without a help page. (rstudio-pro#7063)
 
 #### Posit Workbench
+- Fixed an issue where very large character vectors in the R global environment could make RStudio initialize more slowly. (rstudio-pro#7226)
```

Comment:
Does this change only impact workbench / server-pro, or would it also change open-source Server? Just wondering if the NEWS item should be under Workbench or under RStudio.